src/frontend/utils/updateLiveParticipants.ts

```diff
@@ -0,0 +1,76 @@
+import { updateLiveParticipants } from 'data/sideEffects/updateLiveParticipants';
+import { Participant } from 'types/Participant';
+import { Video } from 'types/tracks';
+
+export const addParticipantAskingToJoin = async (
+  video: Video,
+  participant: Participant,
+) => {
+  const participants = {
+    participantsAskingToJoin: video.participants_asking_to_join.concat([
+      participant,
+    ]),
+    participantsInDiscussion: video.participants_in_discussion,
+  };
+  return await updateLiveParticipants(video, participants);
+};
+
+export const addParticipantToDiscussion = async (
```

Comment:
I see no usage of this function in the project.

Reply:
It was defined in src/frontend/data/stores/useJoinParticipant/index.ts, I thought we want to keep it.
Should I remove it ?

Reply:
As you are not using it, what could be the reason to keep it ?